packages/components-js/style.scss

```diff
@@ -0,0 +1,6 @@
+@use '@porsche-design-system/components-js/styles' as *;
+@import '@porsche-design-system/shared/css/styles.css';
+
+/*body {
```

Comment:
Leftover?

Reply:
I want to keep it here, as reminder that it's easily possible to integrate PDS Styles.